pkgs/top-level/python-aliases.nix

```diff
@@ -78,7 +78,6 @@ mapAliases ({
   amiibo-py = throw "amiibo-py has been removed because the upstream repository was removed"; # Added 2025-01-13
   ansible-base = throw "ansible-base has been removed, because it is end of life"; # added 2022-03-30
   ansible-doctor = throw "ansible-doctor has been promoted to a top-level attribute name: `pkgs.ansible-doctor`"; # Added 2023-05-16
-  ansible-later = throw "ansible-later has been promoted to a top-level attribute name: `pkgs.ansible-later`"; # Added 2023-05-16
```

Comment:
Shouldn't we keep this throw and change its description to `"has been removed due to ..."`?

Reply:
I was thinking that too, but since this has been here for so long I thought that we might as well remove it

Reply:
Oh no what I meant was since the package had been promoted to `pkgs.ansible-later`, we should add an alias for `pkgs.ansible-later` in `pkgs/top-level/aliases.nix` so that the attribute throws during evaluation, in case someone is using `pkgs.ansible-later` somewhere in their derivation.

Reply:
Oh, gotcha! Will do :+1: